UnityProject/Assets/Scripts/Core/Database/ServerData.ServerStatus.cs

```diff
@@ -161,6 +161,10 @@ IEnumerator SendServerStatus()
 				status.CurrentMap = SubSceneManager.ServerChosenMainStation;
 			}
 
+			status.Passworded = !string.IsNullOrEmpty(config.ConnectionPassword);
```

Comment:
== false

Reply:
I can do that but that doesn't seem very easy to understand in a variable declaration

Reply:
thouggh it is coding practice to do == false, 
I think it's fine with stuff like !NoPassword, 
where it is a direct inverse of a bool that is already there